clang/include/clang/Frontend/CASDependencyCollector.h

```diff
@@ -12,8 +12,10 @@
 #include "clang/Frontend/Utils.h"
 #include "llvm/CAS/CASReference.h"
 
-namespace llvm::cas {
```

Comment:
This fails on Windows, but appears to be a warning elsewhere 🤷.

Reply:
Thank you!